draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1146,18 +1151,13 @@ using the "application/x-www-form-urlencoded" format described in the "HTML 4.01
 Specification" [HTML401]. Binary data is base64 encoded [RFC4648] as specified
 in the individual messages.
 
-Clients are configured with a base URL for a log and construct URLs for requests
-by appending suffixes to this base URL. This structure places some degree of
-restriction on how log operators can deploy these services, as noted in
-[RFC7320]. However, operational experience with version 1 of this protocol has
-not indicated that these restrictions are a problem in practice.
+Clients are configured with a log's Base URL (see {{log_parameters}}), and they
+construct URLs for requests by appending suffixes to this Base URL, as described
+in the subsections below.
```

Comment:
I note you left off the sentence about restricting the deployment of the log as a service and it not being an issue based on V1 deployment. Any particular reason other than it not fitting the current structure of the text?

Reply:
I think that sentence was our justification for not having switched to using .well-known.  And so by switching to using .well-known, that sentence became irrelevant.